### DISCUSSION TITLE
老家拍的好看不？

### DISCUSSION THREAD

**Discussion by Z-Century** [share121/inter-knot#161] [其他]
![微信图片_20240809202231](https://github.com/user-attachments/assets/8718a309-8568-4776-8327-e7b76403121d)
![微信图片_20240809202236](https://github.com/user-attachments/assets/45ad6535-8f1b-4086-a47c-3ad7f1872a63)

**Comment by Z-Century** [comment]
好看！！！

**Comment by Hexiaoling457** [comment]
好看！

**Comment by Liangjl-2024** [comment]
哇，真好看，这是在几分街上吗？还是在空洞里拍到的？要真能在空洞里看到这样风景的话我就是死在空洞里也值了！[doge]

**Comment by JingLu170** [comment]
好看

**Comment by yinli166** [comment]
好美的老家
